src/App.css

```diff
@@ -5,13 +5,27 @@ body,
   width: 100vw;
   padding: 0;
   margin: 0;
-  color: white;
   font-size: 100%;
   line-height: normal;
+  color: white;
+}
+
+.screen {
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
You forgot to use CSS variable here

Reply:
I'm using it only at one place. So, did not use variable here.